README.md

```diff
@@ -60,12 +60,33 @@ Project has no opportunity to authorize new users as edX's instance student.
 So you need to do this manually:
 1. Go to admin panel, then to `Edx telegram users` model (table).
 2. Make relationship between Telegram user and edX user in field `student`.
-3. Add telegram `id` and telegram `nickname`.
+3. Add telegram `id` and telegram `username` (optional move, it can be empty - `''`).
```

Comment:
`s/(optional move, it can be empty - `''`)/(optional)/` --- will be fair enough.